<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>600</height>
   </rect>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="mainPage">
    <layout class="QGridLayout" name="gridlayout">
     <item row="2" column="0" colspan="3">
      <layout class="QHBoxLayout" name="chartLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QChartView" name="PBanquierChart">
         <property name="cursor" stdset="0">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QChartView" name="PAssuranceChart"/>
       </item>
       <item>
        <widget class="QChartView" name="PBoursierChart"/>
       </item>
      </layout>
     </item>
     <item row="0" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="AgencyLabel">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Agence</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="agenciesComboBox">
         <item>
          <property name="text">
           <string>Toutes les agences</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Agence de Strasbourg</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Agence de Lille</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Agence de Nice</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Agence de Lyon</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Agence de Paris</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="fromDateLabel">
         <property name="text">
          <string>Du</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="fromDateEdit">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
         <property name="date">
          <date>
           <year>2021</year>
           <month>1</month>
           <day>16</day>
          </date>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="toDateLabel">
         <property name="text">
          <string>Au</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="toDateEdit">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
         <property name="date">
          <date>
           <year>2021</year>
           <month>4</month>
           <day>16</day>
          </date>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="insightLabel">
         <property name="text">
          <string>Aperçu</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="InsightComboBox">
         <item>
          <property name="text">
           <string>Volumes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Chiffre d'affaire (€)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Pourcentage</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="themeCheckBox">
         <property name="text">
          <string>Thème sombre</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="bankPage">
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <layout class="QGridLayout" name="gridlayout1"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>983</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBanque">
    <property name="title">
     <string>Banque</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuBanque"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>themeCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateUI()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toDateEdit</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateUI()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>InsightComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateUI()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fromDateEdit</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateUI()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>agenciesComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateUI()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>485</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateUI()</slot>
 </slots>
</ui>
